Source/Core/Core/PowerPC/JitCommon/JitBase.h

```diff
@@ -116,7 +116,7 @@ class JitBase : public CPUCoreBase
   JitBase();
   ~JitBase() override;
 
-  static const u8* Dispatch() { return g_jit->GetBlockCache()->Dispatch(); }
+  static const u8* Dispatch(JitBase& jit);
```

Comment:
This will break https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp#L127

MOVP2R(X0, reinterpret_cast<void*>(&m_jit));